lib/Gedmo/Tree/Entity/Repository/ClosureTreeRepository.php

```diff
@@ -315,7 +315,13 @@ public function buildTreeArray(array $nodes)
                     $tmp = &$refs[$n['parent_id']][$childrenIndex];
                 }
 
-                $key = count($tmp);
+                if(is_array($tmp) || $tmp instanceof \Countable) {
+                    $key = count($tmp);
+                } elseif(null === $tmp) {
+                    $key = 0;
```

Comment:
```suggestion
                if (null === $tmp) {
                    $key = 0;
                } elseif (is_array($tmp) || $tmp instanceof \Countable) {
                    $key = count($tmp);
```
BTW, I'm not using this extension, so I don't know what possible types can be held in `$tmp`. By instance, I don't know if we should even consider `iterator_count()`.